src/components/Login/Logout.jsx

```diff
@@ -27,7 +28,22 @@ const Logout = () => {
         <label id="labelLogout" htmlFor="btnLogout">
           Click the following logout button to log out of your pod:{' '}
         </label>
-        <LogoutButton onLogout={handleLogout} />
+        <button type="submit" onClick={() => setShowConfirmation(true)}>
+          Logout
+        </button>
+        {showConfirmation ? (
+          <dialog open>
+            <p>Do you want to log out now?</p>
+            <div>
+              <LogoutButton onLogout={handleLogout}>
+                <button>Logout</button>
+              </LogoutButton>
+            </div>
+            <button onClick={() => setShowConfirmation(false)}>Cancel</button>
+          </dialog>
+        ) : (
+          <div></div>
+        )}
```

Comment:
If we combine dialog with showConfirmation state, ternary will not be needed anymore since it'll become just a singular component:

```jsx
<dialog open={showConfirmation}>
  {/* other components here */}
</dialog>
```

Otherwise, if we're keeping it as:

```jsx
{showConfirmation
  ? <dialog open>{/* other components here */}</dialog> 
  : <div></div>
}
```

We can replace the empty div with null

Reply:
That's a great point -- I've updated this to your first suggestion.